hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -379,6 +379,8 @@
   public static final String DFS_PROVIDED_ALIASMAP_TEXT_WRITE_DIR_DEFAULT = "file:///tmp/";
 
   public static final String DFS_PROVIDED_ALIASMAP_LEVELDB_PATH = "dfs.provided.aliasmap.leveldb.path";
+  public static final String DFS_NAMENODE_MOUNT_ACLS_ENABLED = "dfs.namenode.mount.acls.enabled";
```

Comment:
Should we refer to PROVIDED here?

Reply:
Agree. WDYT about `dfs.namenode.provided.acls.enabled`? 
Keep `namenode` since the config is a NN config. 
Replaced `mount` with `provided` so that the config name is consistent.

Reply:
Better than before for sure.
Not sure about the end policy for distinguishing namenode from not.
It should be consistent.